.circleci/config.yml

```diff
@@ -588,6 +588,37 @@ jobs:
           name: Build the template application
           command: cd template/android/ && ./gradlew assembleDebug
 
+  # -------------------------
+  #    JOBS: Test Android RNTester
+  # -------------------------
+  test_android_rntester:
+    executor: reactnativeandroid
+    parameters:
+      use_hermes:
+          type: boolean
+          default: false
+    steps:
+      - checkout
+      - run_yarn
+
+      - run:
+          name: Generate artifacts for Maven
+          command: ./gradlew :ReactAndroid:installArchives
+
+      - when:
+          condition: << parameters.use_hermes >>
+          steps:
+            - run:
+                name: Build RNTester with Hermes
+                command: ./gradlew :packages:rn-tester:android:app:assembleHermesDebug
```

Comment:
Small follow up - can we change this to just:
```
./gradlew :packages:rn-tester:android:app:assembleDebug
```
(or even `assemble`?)

`assembleDebug` will invoke both Hermes and JSC tasks and will run faster than two separate Gradle invocations

Reply:
Sure, I've just opened a PR tackling this https://github.com/facebook/react-native/pull/33042, I went with `assembleDebug` because I don't think we need to assemble both release and debug, but let me know if you prefer using just `assemble` instead

Reply:
`assembleDebug` is fine for now 👍 Thanks for doing it.